README.md

```diff
@@ -87,34 +86,39 @@ Here are the dependencies for the optional features:
 Documentation
 =============
 
-To build documentation for this project, see the documentation 
+To build documentation for this project, see the documentation
 [instructions](docs/contributing/documentation.rst).
 
-Using Katana as a Library 
+Using Katana as a Library
 =========================
 
 There are two common ways to use Katana as a library. One way is to copy this
 repository into your own CMake project, typically using a git submodule. Then
-you can put the following in your CMakeLists.txt:
+you can put the following in your `CMakeLists.txt`:
 
 ```CMake
-add_subdirectory(galois EXCLUDE_FROM_ALL)
+add_subdirectory(katana EXCLUDE_FROM_ALL)
 add_executable(app ...)
 target_link_libraries(app Katana::galois)
 ```
 
 The other common method is to install Katana outside your project and import it
-as a package.
+as a package. If you install the `katana-cpp` Conda package as follows, `Katana` cmake package will be available:
 
-If you want to install Katana, assuming that you wish to install it under
+```Shell
+conda install -c katanagraph/label/dev -c conda-forge katana-python
+```
+
+If you want to build and install Katana, assuming that you wish to install it under
```

Comment:
Alternatively, you can install Katana from source. The following command will build and install Katana into `INSTALL_DIR`: